employee-rostering-gwtui/src/main/java/org/optaweb/employeerostering/gwtui/client/spot/SpotListPanel.java

```diff
@@ -34,20 +35,20 @@
 import org.jboss.errai.ui.shared.api.annotations.EventHandler;
 import org.jboss.errai.ui.shared.api.annotations.ForEvent;
 import org.jboss.errai.ui.shared.api.annotations.Templated;
-import org.optaweb.employeerostering.gwtui.client.common.DataInvalidation;
+import org.optaweb.employeerostering.gwtui.client.common.EventManager;
+import org.optaweb.employeerostering.gwtui.client.common.EventManager.Event;
 import org.optaweb.employeerostering.gwtui.client.common.FailureShownRestCallback;
 import org.optaweb.employeerostering.gwtui.client.common.KiePager;
 import org.optaweb.employeerostering.gwtui.client.common.KieSearchBar;
 import org.optaweb.employeerostering.gwtui.client.pages.Page;
 import org.optaweb.employeerostering.gwtui.client.tenant.TenantStore;
-import org.optaweb.employeerostering.gwtui.client.util.CommonUtils;
 import org.optaweb.employeerostering.gwtui.client.util.PromiseUtils;
 import org.optaweb.employeerostering.shared.spot.Spot;
 import org.optaweb.employeerostering.shared.spot.SpotRestServiceBuilder;
 
 @Templated
 public class SpotListPanel implements IsElement,
-        Page {
+                           Page {
```

Comment:
`Page` should be aligned with `IsElement`. @Christopher-Chianelli I don't want to block the PR with this one but sooner we get Eclipse and IDEA formatters in sync the sooner we'll get clean PRs without unnecessary whitespace changes,

Reply:
@Christopher-Chianelli Thanks for confirmation. Do you think you can fix the Eclipse configuration and submit a PR to `droolsjbpm-build-bootstrap?

Reply:
@yurloc It appears IDEA can use Eclipse formatter, but not the other way around: https://intellij-support.jetbrains.com/hc/en-us/community/posts/206575559-How-to-Export-Code-Formatting-Configurations-to-Eclipse. I cannot guarantee the formatters will be one-to-one, but I try to make it as one-to-one as possible. We should consider adding a format checker in addition to the already existing checkstyle checker (https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/f8000107b3d94c2a05e46beefa0b9cbedd2850c1/pom.xml#L590). See https://github.com/revelc/formatter-maven-plugin (this uses the eclipse formatter, did not find one that uses the IDEA formatter yet).

Reply:
@yurloc Can confirm it is impossible to make `Page` be aligned with `IsElement` in the Eclipse formatter. "Line Wrapping > 'implements' cause > Indentation policy" controls the indent. There are three options: "Default indentation", which causes it to be indented at column 8 (2 standard indents), "Indent on column", which causes it to be indented on "implements" and "Indent by 1", which causes it to be indented at column 4 (1 standard indent). I can still try to make methods chains less bad.